elastic/kibana
# [Monitoring] Fixed server response errors

Resolves https://github.com/elastic/kibana/issues/62843

This was caused because we weren't try/catching our main resolve/reject state

Testing
1. Create a user with an insufficient role to enable monitoring
2. Go to Kibana > Stack Monitoring > Click on `Or, set up with self monitoring`
3. Notice the correct error type/statusCode
4. The same errors can also be visible in the browser's console
